sys-fs/bees/bees-0.11_rc3.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit linux-info systemd toolchain-funcs
+
+DESCRIPTION="Best-Effort Extent-Same, a btrfs dedup agent"
+HOMEPAGE="https://github.com/Zygo/bees"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Zygo/bees.git"
+else
+	MY_PV=${PV/_/-}
+	MY_P=${P/_/-}
+	S=${WORKDIR}/${MY_P}
+
+	SRC_URI="https://github.com/Zygo/bees/archive/v${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64"
```

Comment:
We generally want alphas/betas/rcs to be unkeyworded.

if [[ ${PV} != *_rc* ]] ; then ... fi

Reply:
Thanks, should be fixed now.